conf/layer.conf

```diff
@@ -0,0 +1,13 @@
+# We have a conf and classes directory, add to BBPATH
+BBPATH .= ":${LAYERDIR}"
+
+# We have recipes-* directories, add to BBFILES
+BBFILES += "${LAYERDIR}/recipes-*/*/*.bb \
+            ${LAYERDIR}/recipes-*/*/*.bbappend"
+
+BBFILE_COLLECTIONS += "oe-scipy"
+BBFILE_PATTERN_oe-scipy = "^${LAYERDIR}/"
+BBFILE_PRIORITY_oe-scipy = "6"
+
+LAYERDEPENDS_oe-scipy = "core"
```

Comment:
You should add `openembedded-layer` and `meta-python` to the `LAYERDEPENDS` variable.